test.js

```diff
@@ -6,78 +6,83 @@ const prettier = require('prettier');
  * @param {string} code
  * @param {prettier.Options} options
  */
-const prettify = (code, options) =>
-	prettier.format(code, { filepath: 'file.ts', parser: 'typescript', plugins: ['.'], ...options });
+const prettify = (code, options) => prettier.format(code, { plugins: ['.'], filepath: 'file.ts', ...options });
 
-test('sorts imports', (t) => {
-	const code = `
-		import { foo, bar } from "foobar"
+const prettierOptions = [{ parser: 'babel' }, { parser: 'babel-ts' }, { parser: 'typescript' }];
 
-		export const foobar = foo + bar
-	`;
+prettierOptions.forEach((options) => {
```

Comment:
Hm, wondering whether this is a good use case for a "macro" 🤔 

https://github.com/avajs/ava/blob/main/docs/01-writing-tests.md#reusing-test-logic-through-macros

Reply:
I wondered that too, but I'm not very familiar with Ava. The documentation wasn't really clear about whether or not to use macros for multiple tests at a time, or if they were just for reusing individual tests.

With macros you'd still have to write one line per test + test case without a loop (so 18 lines here).

I based my approach on the thread here: https://github.com/avajs/ava/issues/1967.